frontend/src/scenes/events/eventsTableLogic.ts

```diff
@@ -328,6 +339,13 @@ export const eventsTableLogic = kea<eventsTableLogicType<ApiError, EventsTableLo
                 isNext: !!nextParams,
             })
 
+            if (
+                apiResponse.results.length === 0 &&
+                dayjs(values.miniumumQueryDate).diff(dayjs(nextParams?.after || '1980-01-01'), 'minute') > 5
```

Comment:
this took some kreplits to parse, (if I'm right) if there are no API results and we're not close to the end of the pages of data in minimum query date range then load across the whole N months of minimum query date range to see if we get a result then... so there's a lot to unpack

I'd be tempted to give it a name or a comment to save someone else having to translate it in future

Reply:
makes lots of sense, just realised this while reviewing myself, added a comment, lmk if that's sound / should change further!